template-resourcegroup.yaml

```diff
@@ -90,4 +90,5 @@ Resources:
     #   TapeCapacity: 0
 
   # Other resources if you have any...
-
+  #   <SERVICE NAME>:
```

Comment:
No, the "other resources" comment refers to having additional _entire resources_ e.g. everything from line 11 to here. Sounds like that needs to be clarified here -- can you suggest anything?

Reply:
Oops, that's my own fault for not reading closely enough. New commit incoming